dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/programrule/implementers/RuleActionExecutor.java

```diff
@@ -25,20 +25,21 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.hisp.dhis.tracker.programrule.implementers.enrollment;
+package org.hisp.dhis.tracker.programrule.implementers;
 
 import java.util.Optional;
 
 import org.hisp.dhis.tracker.bundle.TrackerBundle;
-import org.hisp.dhis.tracker.domain.Enrollment;
 import org.hisp.dhis.tracker.programrule.ProgramRuleIssue;
 
-public interface RuleActionExecutor
+public interface RuleActionExecutor<T>
 {
+    String getField();
```

Comment:
I added the javadoc but this is still a controversial topic.
I would probably add it as a point for next Design meeting, I am not sure if this is needed because of a bug in validation when creating a Program Rule or if it is supposed to be like this